modules/axhal/src/arch/aarch64/hv/exception.rs

```diff
@@ -1,51 +1,29 @@
 use core::arch::global_asm;
-use core::sync::atomic::{AtomicUsize, Ordering};
-
-use aarch64_cpu::registers::{ESR_EL2, FAR_EL2};
 use tock_registers::interfaces::*;
 
-use memory_addr::VirtAddr;
-use page_table_entry::MappingFlags;
-
-use super::exception_utils::*;
+global_asm!(include_str!("exception.S"));
 use crate::arch::TrapFrame;
 
-global_asm!(include_str!("exception.S"));
+use core::sync::atomic::{AtomicUsize, Ordering};
 
-type ExceptionHandler = fn(&mut TrapFrame);
+type VmExitHandler = unsafe extern "C" fn();
 
-const EMPTY_HANDLER_VALUE: AtomicUsize = AtomicUsize::new(0);
+#[no_mangle]
+static mut LOWER_AARCH64_SYNCHROUNOUS_HANDLER: VmExitHandler = dummy_vmexit_handler;
 
-static LOWER_AARCH64_HANDLERS: [AtomicUsize; MAX_EXCEPTION_COUNT] =
-    [EMPTY_HANDLER_VALUE; MAX_EXCEPTION_COUNT];
+#[no_mangle]
+static mut LOWER_AARCH64_IRQ_HANDLER: VmExitHandler = dummy_vmexit_handler;
 
-const MAX_EXCEPTION_COUNT: usize = 64;
+unsafe extern "C" fn dummy_vmexit_handler() {}
 
 #[allow(dead_code)]
-pub fn register_exception_handler_aarch64(
-    exception_class: usize,
-    handler: ExceptionHandler,
-) -> bool {
-    if exception_class < MAX_EXCEPTION_COUNT {
-        LOWER_AARCH64_HANDLERS[exception_class].store(handler as usize, Ordering::SeqCst);
-        return true;
-    }
-    return false;
+pub unsafe fn register_lower_aarch64_synchronous_handler(handler: VmExitHandler) {
```

Comment:
This is a totally invasive modification for arceos; it's best to have xxx review it.

Let's ask @equation314 for suggestions.